module/entities/helper.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * name : helper.js
+ * author : Ankit Shahu
+ * created-date : 16-Jan-2024
+ * Description : entities helper functionality.
+ */
+
+const userProfileService = require(GENERICS_FILES_PATH + "/services/users");
+
+module.exports = class entitieHelper {
+
+ /**
+   * update registry in entities.
+   * @method
+   * @name listByLocationIds
+   * @param {Object} locationIds - locationIds
+   * @returns {Object} entity Document
+   */
+
+    static listByLocationIds(locationIds) {
+        return new Promise(async (resolve, reject) => {
+            try {
+                //if not uuid considering as location code- for school.
+                let locationDeatails = UTILS.filterLocationIdandCode(locationIds,);
+                //set request body for learners api
+                let entityInformation = [];
+                let formatResult = true;
+
+                if ( locationDeatails.ids.length > 0 ) {
+                    let bodyData = {
+                        "id" : locationDeatails.ids
+                    } 
+                    let entityData = await userProfileService.locationSearch( bodyData, formatResult );
+                    if ( entityData.success ) {
+                        entityInformation =  entityData.data;
+                    }
+                }
+
+                if ( locationDeatails.codes.length > 0 ) {
+                    let bodyData = {
+                        "code" : locationDeatails.codes
+                    } 
+                    let entityData = await userProfileService.locationSearch( bodyData, formatResult );
```

Comment:
@Prajwal17Tunerlabs add a key in env to understand which entity service we need to call. according to that key call the entity service